deploy/docker/cp-dts-log-checker/dts-log-checker.py

```diff
@@ -0,0 +1,187 @@
+    <html xmlns="http://www.w3.org/1999/xhtml">
+    <head>
+        <meta http-equiv="Content-Type" content="text/html; charset=utf-8" />
+        <style>
+            table,
+            td {{
+                border: 1px solid black;
+                border-collapse: collapse;
+                padding: 5px;
+            }}
+        </style>
+    </head>
+    <body>
+        <p>Dear user,<br>
+        *** This is a system generated email, do not reply to this email ***<br>
+        <b>DTS log files with errors:</b><br>
+        {}<br>
+        Best regards, 
+        EPM-AWS-Dev Platform<br>
+    </body>
+    </html>
+        """
+
+    return body
+
+
+def generate_table(files_with_errors):
+    table = """<table>
+    <tr>
+		<td><b>Files</b></td>
+        <td><b>Url</b></td>
+	</tr>"""
+    for file in files_with_errors:
+        table += "<tr>" + \
+            "<td>{}</td>".format(file) + "<td>{}</td>".format(str(dts_log_url) + \
+            os.path.basename(file) + '&contentDisposition=ATTACHMENT') + "</td>"
+
+    table += "</table>"
+    return table
+
+
+def retrieve_timestamp(str):
+    match_str = re.search(
+        '(\d{4})-(\d{2})-(\d{2}) (\d{2}):(\d{2}):(\d{2}).(\d{3})', str)
+    if match_str is not None and match_str.group():
+        return datetime.strptime(match_str.group(), "%Y-%m-%d %H:%M:%S.%f")
+    else:
+        logger.info('There is no applicable date')
+
+
+def retrieve_last_results_info():
+    if os.path.exists(dts_file) and os.path.getsize(dts_file) > 0:
+        global need_to_seek
+        need_to_seek = True
+        with open(dts_file) as f:
+            file_row = f.read()
+            dates = file_row.split(',')
+            global script_runtime
+            script_runtime = retrieve_timestamp(dates[0])
+            global last_row_number
+            last_row_number = dates[1]
+        for file in dts_logs_files:
+            mtime = os.path.getmtime(file)
+            last_modification_date = datetime.fromtimestamp(mtime)
+            if last_modification_date > script_runtime:
+                changed_files.append(file)
+    else:
+        logger.info('There is no information about previous results')
+
+
+def file_contains_error(filename):
+    with open(filename, 'r') as f:
+        for line in f:
+            if re.search(pattern, line):
+                return True
+
+
+last_row_number = ''
+script_runtime = ''
+changed_files = []
+need_to_seek = False
+
+retrieve_last_results_info()
+files_with_errors = set()
+
+if need_to_seek and not changed_files:
+    logger.info('No new files to check')
+
+elif need_to_seek and changed_files:
+    sorted_changed_files = sorted(
+        changed_files, key=os.path.getmtime, reverse=False)
+    logger.info('There are new files, a check is needed')
+
+    oldest_file = open(sorted_changed_files[0], 'r')
+    count = 0
+    for line in oldest_file:
```

Comment:
I believe we can unite logic for file that was checked last time and new files, that are need to be checked fully.
For this case we should change method ` def file_contains_error(filename):` as ` def file_contains_error(filename, line_to_start):`

and body as:
```
        if count < int(line_to_start):
            count += 1
            continue
        if re.search(pattern, line):
           return True, count
        return False, count
```